include/swift/Subsystems.h

```diff
@@ -276,6 +283,20 @@ namespace swift {
   bool performLLVM(IRGenOptions &Opts, ASTContext &Ctx,
                    llvm::Module *Module);
 
+  /// Run the LLVM passes. In multi-threaded compilation this will be done for
+  /// multiple LLVM modules in parallel.
+  bool performLLVM(IRGenOptions &Opts, DiagnosticEngine *Diags,
+                   llvm::sys::Mutex *DiagMutex,
+                   llvm::GlobalVariable *HashGlobal,
+                   llvm::Module *Module,
+                   llvm::TargetMachine *TargetMachine,
+                   version::Version const& effectiveLanguageVersion,
```

Comment:
Nitpick: we usually write this `const version::Version &effectiveLanguageVersion`.